src/ngraph/op/experimental/quantized_conv_bias.cpp

```diff
@@ -90,3 +90,146 @@ shared_ptr<Node> op::QuantizedConvolutionBias::copy_with_new_args(const NodeVect
                                                          new_args.at(3),
                                                          m_with_relu));
 }
+
+op::QuantizedConvolutionBiasAdd::QuantizedConvolutionBiasAdd(const shared_ptr<Node>& data_batch,
```

Comment:
It feels like overkill having a new op here.  @nbpatel tells me we support the same op on the CPU backend with the aid of pattern matching for f32.  The same should be true for int8.  nGraph has a reduced instruction set architecture.  We've made some accommodations in the experimental directory given our immediate goals but (so far) this is just for TensorFlow, just for CPU and just for the few models we are targeting in the near term.  How many models does this op set enable?  Two?  I'm concerned that if we keep expanding the experimental ops the solution will be untenable - not for  TensorFlow and the CPU backend, of course, but as we expand to other frameworks and backends.  I'm not arguing the builder for the same op... we need that.  I'm saying the builder should build a subgraph in this case, not return an op and the CPU backend should pattern match in the right MKL-DNN kernel.